packages/wdio-allure-reporter/src/utils.js

```diff
@@ -14,13 +14,18 @@ export const getTestStatus = (test, config) => {
         return testStatuses.FAILED
     }
 
+    if (test.error.name && test.error.message) {
+        const message = test.error.message.trim()
+        return (test.error.name === 'AssertionError' || message.startsWith('Expect'))  ? testStatuses.FAILED : testStatuses.BROKEN
```

Comment:
How certain are we that every message starts with `Expect`?

Reply:
I'm sure that every message starts with `Expect` , please see code in the `expect-webdriverio` package :   https://github.com/webdriverio/expect-webdriverio/blob/d27f4a30f6b1866e8847ce9df26fa613d54e8ed6/src/util/formatMessage.ts#L89